src/current/v23.1/cluster-setup-troubleshooting.md

```diff
@@ -441,6 +441,16 @@ CockroachDB's built-in disk stall detection works as follows:
 
 - During [node liveness heartbeats](#node-liveness-issues), the [storage engine]({% link {{ page.version.version }}/architecture/storage-layer.md %}) writes to disk as part of the node liveness heartbeat process.
 
+#### Disk usage is different across nodes in the cluster
+
+This is expected behavior.
+
+CockroachDB uses [load-based replica rebalancing]({% link {{ page.version.version }}/architecture/replication-layer.md %}#load-based-replica-rebalancing) to spread [replicas]({% link {{ page.version.version }}/architecture/overview.md %}#architecture-replica) across the nodes in a cluster.
+
+The rebalancing criteria for load-based replica rebalancing do not include 'disk usage per node'. Not all [ranges]({% link {{ page.version.version }}/architecture/overview.md %}#architecture-range) are the same size. Larger and smaller ranges balance each other out on average, so disk usage differences across nodes should be relatively small.
+
+However, in some cases a majority of the largest (or smallest) ranges are on one node. Normally that shouldn't be a problem with [sufficiently provisioned storage]({% link {{ page.version.version }}/recommended-production-settings.md %}#storage). If it is causing issues, please [contact Support]({% link {{ page.version.version }}/support-resources.md %}) so they can guide you on how to manually rebalance your cluster's disk usage.
```

Comment:
Do we mention that there are guards against overfilling a disk somewhere?

The disk usage isn't being balanced, however we will move things around if we hit the thresholds described in https://github.com/cockroachdb/cockroach/pull/97409.

It could be helpful to clarify that, so that users don't see this as a signal we have no protection against full disks.

Reply:
> Do we mention that there are guards against overfilling a disk somewhere?

I searched the docs and it appears we do not mention any guards against overfilling. We mention [provisioning sufficient capacity](https://www.cockroachlabs.com/docs/v23.1/recommended-production-settings#storage), [monitoring capacity to avoid going over 80%](https://www.cockroachlabs.com/docs/v23.1/common-issues-to-monitor#storage-capacity) and [using emergency ballast files](https://www.cockroachlabs.com/docs/v23.1/cluster-setup-troubleshooting#automatic-ballast-files)

> The disk usage isn't being balanced, however we will move things around if we hit the thresholds described in https://github.com/cockroachdb/cockroach/pull/97409.

Thanks for that info! I have updated this section to add a bulleted list with some information about how the allocator helps protect against full disks - PTAL

Re: the specific cluster settings in https://github.com/cockroachdb/cockroach/pull/97409, they don't appear to be set as "public" and therefore don't appear in the [cluster settings docs page](https://www.cockroachlabs.com/docs/stable/cluster-settings), so I haven't mentioned them explicitly.  I can mention them here if hiding them from the auto-generated docs was not intentional.  We can also leave it like this with a prose description and let the settings be revealed to users through a support interaction if we'd prefer folks not touch them otherwise.

Reply:
The latter option sounds best. These settings are useful but don't have significant guardrails, so the prose option is good.